base/math.jl

```diff
@@ -833,45 +833,12 @@ minmax(x::T, y::T) where {T<:AbstractFloat} = min(x, y), max(x, y)
 
 _isless(x::Float16, y::Float16) = signbit(widen(x) - widen(y))
 
```

Comment:
I think this method can be removed too.  As far as I can tell it's not used anywhere (`_isless` is used in `min`/`max` method above but they don't apply to `Float16`).  The `_isless` function also appears to be used in https://github.com/JuliaLang/julia/blob/664528adac261df806f291493593f63b66ff169c/base/mpfr.jl#L886-L899 but also that one doesn't apply to `Float16`
```suggestion
```

Reply:
yeah this code appears to be from when we assumed that no one supported Float16.